docs/jwst/outlier_detection/outlier_detection_tso.rst

```diff
@@ -0,0 +1,41 @@
+.. _outlier-detection-tso:
+
+Outlier Detection for TSO Data
+==============================
+
+Time-series observations (TSO) result in input data stored as a 3D CubeModel
+where each plane in the cube represents a separate integration without changing the
+pointing.  Normal imaging data benefit from combining all integrations into a
+single image. TSO data's value, however, comes from looking for variations from one
+integration to the next.  The outlier detection algorithm, therefore, gets run with 
+a few variations to accomodate the nature of these 3D data.
+
+#. Input data is converted into a CubeModel (3D data array) if a ModelContainer
+   of 2D data arrays is provided.
+
+#. The median image is created without resampling the input data
+
+   * The median image is created by combining all planes in the 
+     ModelContainer pixel-by-pixel using a rolling-median algorithm, in order
```

Comment:
Aren't the data planes that're being median filtered coming from a single 3D `CubeModel`, as opposed to a `ModelContainer`? In fact the top bullet above says that data from a `ModelContainer` get copied into a 3D `CubeModel` for the remainder of the processing. Hence this should say "... all planes in the CubeModel ..."